content/auth/token.textile

```diff
@@ -569,6 +689,63 @@ $base64Signature = base64_encode($signature);
 $jwt = $base64Header . '.' . $base64Claims . '.' . $base64Signature;
 ```
 
+```[go]
```

Comment:
This is a code representing server side construction of `jwt` token. 
I would also recommend adding client side code that uses `jwt` token using `authCallback`.
You can refer to https://github.com/ably/ably-dotnet?tab=readme-ov-file#using-the-token-auth-at-client-side

Reply:
closing in favor of https://github.com/ably/docs/pull/2201#discussion_r1662291666